code_schemes/rqas/giz/s01e01.json

```diff
@@ -0,0 +1,206 @@
+{
+    "SchemeID": "Scheme-de4d4637",
+    "Name": "S01E01",
```

Comment:
Should we start adding the project name here too e.g. "GIZ S01E01"? At the moment the only way to tell which project this code scheme is for is by looking at its location on disk.